website/docs/docs/core/connect-data-platform/redshift-setup.md

```diff
@@ -65,23 +114,23 @@ company-name:
       port: 5439
       dbname: analytics
       schema: analytics
+      connect_timeout: None
+      sslmode: prefer
+      role: None
+      ra3_node: true 
+      autocommit: true 
```

Comment:
thinking out loud. I almost wonder if these optional parameters belong a separate section rather than duplicated in the example profiles.

in my head, the example `profiles.yml` is the basic thing you can copy and paste that has bare minimum to get started with your chosen auth method. the optional params are something you reach for if you need something specific? idk

relatedly, is `role` used for db auth?

Reply:
role is used for both db and iam auth - it is used to run 'set role', which sets the user for current session.

Imo having one profiles.yml including both required and optional params seems cleaner than having multiple sections for additional configs? thanks for bringing this up though - I've added all the optional params under the comment   `# Optional Redshift configs`. Users get to copy and paste whatever thats needed from the optional configs, do you think this structure makes sense? Also, if you check out the deployed preview for this PR, the profiles.yml actually isn't duplicated. Its a different profiles.yml that shows up under the tab `IAM` and `database`